libs/sdk-ui-ext/src/internal/components/configurationPanels/LineChartBasedConfigurationPanel.tsx

```diff
@@ -38,7 +42,14 @@ export default class LineChartBasedConfigurationPanel extends BaseChartConfigura
                             pushData={pushData}
                             properties={properties}
                             isDisabled={controlsDisabled}
-                            defaultValue={false}
+                            defaultValue={"auto"}
```

Comment:
this change of default value is on purpose?

Reply:
By accident, I'll revert it.